spec/expression.dd

```diff
@@ -2335,20 +2329,33 @@ void foo()
         writeln("not satisfied");
 }
 -------------
-        )
 
-        $(LI $(D is $(LPAREN)) $(I Type) $(D ==) $(I TypeSpecialization) $(D $(RPAREN))$(BR)
+        $(H5 $(LNAME2 is-type-equal, $(D is $(LPAREN)) $(I Type) $(D ==) $(I TypeSpecialization) $(D $(RPAREN))))
```

Comment:
These seem (for me at least) to get listed without a number (the trailing paragraph is then numbered)

Reply:
Yes, I think the css code for that is in the `BODY_PREFIX` macro in `spec/spec.ddoc`. I'm not sure how to update it for h5.